web/pdf_document_properties.js

```diff
@@ -79,9 +79,14 @@ class PDFDocumentProperties {
           contentDispositionFilename || getPDFFileNameFromURL(this.url),
           this._parseFileSize(this.maybeFileSize),
           this._parseDate(info.CreationDate),
-          this._parseDate(info.ModDate)
+          this._parseDate(info.ModDate),
+          this.pdfDocument.getPageSizeInches().then((pageSizeInches) => {
+            return this._parsePageSize(pageSizeInches);
+          }),
+
         ]);
-      }).then(([info, metadata, fileName, fileSize, creationDate, modDate]) => {
+      }).then(([info, metadata, fileName, fileSize,
+                creationDate, modDate, pageSize]) => {
```

Comment:
use `[ pageSizeInch, pageSizeMM ]` instead of `pageSize`; or deconstruct in such way latter.